data/assets/gallery-image-stylesheet.scss

```diff
@@ -139,3 +139,20 @@ h1 {
         font-family: $support-font-composite;
     }
 }
+
+/* Smartphones (portrait and landscape) */
+@media only screen 
+and (min-device-width: 320px) 
+and (max-device-width: 719px) {
+    html {
+        font-size:16px;
```

Comment:
Shouldn't we use the mixin for this?

@include responsive-elements($body-size: 16px);